app/views/components/addToList.scala.html

```diff
@@ -15,13 +15,13 @@
             <dd class="hmrc-add-to-a-list__change">
                 <a class="govuk-link" href="@item.changeUrl">
                     <span aria-hidden="true">@messages("site.change")</span>
-                    <span class="govuk-visually-hidden">@messages(changePrefix, item.name)</span>
+                    <span class="govuk-visually-hidden">@messages(changePrefix, countryFromName.getOrElse(item.name), item.name)</span>
```

Comment:
In scenarios where we dont have a countryFrom such as salesFromNIListView then it will show the same country twice in the message. IE: Sales from $CountryTo to $CountryTo. You will need to switch it to what we discussed late on Friday. By checking if the value is there for countryFrom, if it is, use the two parameter version of the message if it isnt then just the one

Reply:
Wrapped condition around spans